test/odbc/psqlodbc/psqlodbc_tests_common.h

```diff
@@ -140,6 +140,20 @@ void verifyValuesInObject(ServerType serverType, const string &objectName, const
  * @param expectedLen A vector containing the expected length of all data in the object. 
  * @param pkStartingValue Optional. The primary key value the object starts incrementing at. The default value is 0.
 */
+
+void testStringFunctions(ServerType serverType, const string &tableName, const vector<string> &operationsQuery, 
+vector<vector<string>> &expectedResults, const vector<string> &pk, const string &colName);
+/**
+ * Verify if the operators in query could work for the data in table
+ * 
+ * @param serverType The ODBC driver type to create the connection against. 
+ * @param tableName The table stored the data
+ * @param operationQuery the query that stored all the operators need to be tested
+ * @param expectResults the expect result after applied the string operators
+ * @param pk  the primary key of the table
+ * @param colName  the column name of the table
+ */
```

Comment:
These should be above the function. Furthermore, you seperated the documentation of verifyValuesInObject by putting this function in there. The comments in lines 127 - 141 iof this file should have been  a part of the function on line 158.  Please fix it